waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -280,7 +301,7 @@
                  :k8s/user run-as-user
                  :log-directory (log-dir-path run-as-user restart-count)
                  :port port0
-                 :service-id (k8s-object->service-id pod)
+                 :service-id service-id
```

Comment:
Why did this change?

Reply:
This changed because I wanted to remove the redundant call to function due to a new logic few lines above but I already changed it back since logic changed since that.